core/domain/suggestion_services.py

```
@@ -97,11 +97,11 @@
 # email.
 MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_REVIEWER: Final = 5
 
-SUGGESTION_TRANSLATE_CONTENT_HTML: Callable[
+SUGGESTION_TRANSLATE_CONTENT_HTML: Callable[  # pylint: disable=invalid-name
     [suggestion_registry.SuggestionTranslateContent], str
 ] = lambda suggestion: suggestion.change_cmd.translation_html
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `SUGGESTION_TRANSLATE_CONTENT_HTML` as a `def`

(E731)